.github/workflows/android.yml

```
@@ -0,0 +1,41 @@
+name: Android Build
```

Also change name of the file to be same 
```suggestion
name: sample-android-build.yml
```

---

Accepted and pushed